src/components/ContactCard/ContactFieldInput.jsx

```diff
@@ -11,26 +10,12 @@ const getInputComponent = inputType =>
 
 class ContactFieldInput extends React.Component {
   state = {
-    hasBeenFocused: false,
-    hasFocus: false
+    hasBeenFocused: false
   }
 
   onFocus = () => {
     this.setState({
-      hasBeenFocused: true,
-      hasFocus: true
```

Comment:
It doesn't seem that the `hasFocus` prop is relevant anymore. Not quite sure though.

Reply:
it seems that it was there only to style the input. Seems fine for me